src/Interfaces.sol

```diff
@@ -155,6 +161,8 @@ contract Interfaces is Admin, Initializable {
     _chainIdMap[ChainIdType.HOLOGRAPH][7][ChainIdType.LAYERZERO] = 11;
     // rinkeby
     _chainIdMap[ChainIdType.HOLOGRAPH][4000000001][ChainIdType.LAYERZERO] = 10001;
+    // goerli
+    _chainIdMap[ChainIdType.HOLOGRAPH][4000000011][ChainIdType.LAYERZERO] = 10021;
```

Comment:
Same with this. Just wondering where these id numbers are coming from or if you have some kind of system for adding new ones

Reply:
Se previous question's answer.